alpaca_trade_api/common.py

```diff
@@ -11,23 +11,25 @@ def get_data_url():
         'APCA_API_DATA_URL', 'https://data.alpaca.markets').rstrip('/')
 
 
-def get_credentials(key_id=None, secret_key=None):
+def get_credentials(key_id=None, secret_key=None, oauth=None):
+    oauth = oauth or os.environ.get('APCA_API_OAUTH')
```

Comment:
OAUTH_TOKEN maybe? @Neal opinion?

Reply:
I agree, OAUTH_TOKEN sounds better. Maybe also prefix it with APCA?